roles/python/tasks/python3.yml

```diff
@@ -1,11 +1,12 @@
 ---
 
+- name: user pip packages
+  pip: name={{item}} state=present executable=pip
+  become: true
+  with_items: '{{pip_packages}}'
```

Comment:
What is pip needed for? Seems we install all python stuff with conda before.

Reply:
resolved